tt_metal/python_env/requirements-dev.txt

```diff
@@ -23,6 +23,8 @@ datasets==2.9.0
 torch==2.2.1.0+cpu
 networkx==3.1
 torchvision==0.17.1+cpu
+torchmetrics==1.3.1
```

Comment:
Please run fast dispatch post-commit, build all C++ binaries, and eager package on main and assert their correctness.

Reply:
The mentioned CIs passed and here are the corresponding links -

- [ all - Fast dispatch post-commit main build ](https://github.com/tenstorrent-metal/tt-metal/actions/runs/8078596332)
- [Build C++ binaries with all configs](https://github.com/tenstorrent-metal/tt-metal/actions/runs/8078604735)
- [eager - Package on main](https://github.com/tenstorrent-metal/tt-metal/actions/runs/8078612614)